VisualStudio.gitignore

```diff
@@ -206,3 +206,12 @@ FakesAssemblies/
 
 # Visual Studio 6 workspace options file
 *.opt
+
+# Visual Studio Lightswitch build output
+**/*.HTMLClient/GeneratedArtifacts
+**/*.DesktopClient/GeneratedArtifacts
+**/*.DesktopClient/ModelManifest.xml
+**/*.Server/GeneratedArtifacts
+**/*.Server/ModelManifest.xml
+**/*.Server/generated.parameters.xml
```

Comment:
I can't confirm that this file should be ignored, and the VS 2015 default .gitignore does not mention it. You may want to omit it for now.